meta-lmp-bsp/recipes-security/optee/optee-os-fio_3.12.0.bbappend

```diff
@@ -1,8 +1,32 @@
+OPTEEMACHINE_apalis-imx6 = "imx-mx6qapalis"
 OPTEEMACHINE_qemuarm64 = "vexpress-qemu_armv8a"
+OPTEEMACHINE_imx6ullevk = "imx-mx6ullevk"
+OPTEEMACHINE_imx8mqevk = "imx-mx8mqevk"
+OPTEEMACHINE_imx8mmevk = "imx-mx8mmevk"
 OPTEEMACHINE_uz = "zynqmp-zcu102"
 
-EXTRA_OEMAKE_append_qemuarm64 = " \
-    CFG_RPMB_FS=y CFG_RPMB_WRITE_KEY=y \
```

Comment:
Can you keep the qemuarm64 related changes? I added those as tee-supplicant will use emulation by default on qemuarm64, and if we enable RPMB we are then allowed to also test fiovb and sks writing on RPMB.

Reply:
Accidently deleted it.
btw, never thought that QEMU support RPMB. It does?

Reply:
If you build tee-supplicant with RPMB_EMU, it ends up storing the data under /var/tee instead of using the actual RPMB device. This is just when running on linux, won't work on u-boot unless emulation is also added there.